### DISCUSSION TITLE
v2.66.0

### DISCUSSION THREAD

**Discussion by nunocoracao** [nunocoracao/blowfish#1441] [Announcements]
## What's Changed
* ⚙️ Update Hugo supported version by @github-actions in https://github.com/nunocoracao/blowfish/pull/1421
* ⚙️ Update Hugo supported version by @github-actions in https://github.com/nunocoracao/blowfish/pull/1431
* 📌 Bump @headlessui/react from 1.7.18 to 1.7.19 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1420
* 📌 Bump puppeteer from 22.6.3 to 22.6.5 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1419
* 📌 Bump prettier-plugin-tailwindcss from 0.5.13 to 0.5.14 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1418
* 📌 Bump puppeteer from 22.6.5 to 22.7.0 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1435
* 🎏 translate hosting-deployment && 202310-blowfish-tutorial to zh-cn by @Weaxs in https://github.com/nunocoracao/blowfish/pull/1415
* 🌐 Improve Vietnamese translation by @bonniss in https://github.com/nunocoracao/blowfish/pull/1428
* 🔖 v2.66.0  by @nunocoracao in https://github.com/nunocoracao/blowfish/pull/1412

## New Contributors
* @bonniss made their first contribution in https://github.com/nunocoracao/blowfish/pull/1428

**Full Changelog**: https://github.com/nunocoracao/blowfish/compare/v2.65.0...v2.66.0

<hr /><em>This discussion was created from the release <a href='https://github.com/nunocoracao/blowfish/releases/tag/v2.66.0'>v2.66.0</a>.</em>

**Comment by anoduck** [comment]
@nunocoracao It's incredible. Seriously, I am floored.

**Reply by nunocoracao** [reply]
Thanks @anoduck
